modules/ROOT/pages/scheduler-concept.adoc

```diff
@@ -93,43 +234,47 @@ Note that the Scheduler component also supports Quartz Scheduler special charact
 * `L`: Last day of the week or month, or last specific day of the month
   (such as `6L` for the last Saturday of the month).
 * `W`: Weekday, which is valid in the month and day-of-the-week fields.
-* `#`: Nth day of the month. For example, `#3` is the third day of the month.
+* `#`: "nth" day of the month. For example, `#3` is the third day of the month.
 
 //source info: +http://www.quartz-scheduler.org/documentation/quartz-2.x/tutorials/crontrigger.html+
 
 This example logs the message "hello" every second:
 
 [source,XML,linenums]
 ----
-<flow name="cronFlow" doc:id="ae257a5d-6b4f-4006-80c8-e7c76d2f67a0" >
-  <scheduler doc:name="Scheduler" doc:id="e7b68ccb-c6d8-4567-87af-aa7904a50359" >
+<flow name="cronFlow" >
+  <scheduler doc:name="Scheduler" >
     <scheduling-strategy >
-      <cron expression="* * * * * ?" timeZone="America/Los_Angeles"/>
+      <cron expression="* * * * * ?" />
     </scheduling-strategy>
   </scheduler>
-  <logger level="INFO" doc:name="Logger" doc:id="e2626dbb-54a9-4791-8ffa-b7c9a23e88a1" message='"hello"'/>
+  <logger level="INFO" doc:name="Logger" message='"hello"'/>
 </flow>
 ----
 
 Note that there are a number of free online tools that can help you build Cron
 expressions.
 
-=== Time Zones (timeZone)
+== Examples in Anypoint Exchange
+
+Several examples that you can find in https://www.mulesoft.com/exchange/[Anypoint Exchange^] show use cases for a Scheduler component:
```

Comment:
Suggestion: Several examples in https://www.mulesoft.com/exchange/[Anypoint Exchange^] show use cases for the Scheduler component:

Reply:
done